Rakefile

```diff
@@ -1,9 +1,9 @@
-require 'rake'
-require 'rake/testtask'
-
-Rake::TestTask.new do |t|
-   t.libs << 'test'
-end
-
-desc "Run tests"
-task :default => :test
+require 'rake'
+require 'rake/testtask'
+
+Rake::TestTask.new do |t|
+   t.libs << 'test'
+end
+
+desc "Run tests"
+task :default => :test
```

Comment:
What changed here? Were the line endings not Unix?

Reply:
That's what I assume. I didn't double check.